addons/medical_engine/XEH_preInit.sqf

```diff
@@ -28,6 +28,15 @@ if (isNil QUOTE(FATAL_SUM_DAMAGE_WEIBULL_K) || isNil QUOTE(FATAL_SUM_DAMAGE_WEIB
     FATAL_SUM_DAMAGE_WEIBULL_L = _x1 / _b1^(1/FATAL_SUM_DAMAGE_WEIBULL_K);
 };
 
+// Cache for armour values of equipped items (vests etc)
+if (isNil QGVAR(armourCache_items)) then {
+    GVAR(armourCache_items) = false call CBA_fnc_createNamespace;
+};
+// Separate cache for uniforms because they work differently
+if (isNil QGVAR(armourCache_uniforms)) then {
+    GVAR(armourCache_uniforms) = false call CBA_fnc_createNamespace;
+};
```

Comment:
I don't see a point for these `isNil`s. the caches should just be created without a check.

Reply:
You're right. Fixed.